torch/_dynamo/variables/nn_module.py

```diff
@@ -455,12 +465,28 @@ def gen_source(source, name):
             )
         elif name == "__getitem__":
             assert not kwargs and len(args) == 1
-            assert type(module).__getitem__ in (
+            inline_supported = (
```

Comment:
This is not correct, all these listed here are implemented in dynamo, the rest are supported by inlining. You can just use ```if type(module).__getitem__ not in (...)```.

Reply:
I am bit confused here, the inline_supported is used as the list in the if statement. Thought its easier to read that way:
```
if type(module).__getitem__ not in inline_supported:
```
am i missing something?

Reply:
Oh, actually these are dynamo builtin supported rather than inline supported, the else condition is inline supported(where you call ```inline_user_function_return```).

Reply:
ow, got it now, thanks!